puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/vars/hammer-cli-master-release.groovy

```diff
@@ -0,0 +1,6 @@
+def git_url = 'https://github.com/theforeman/hammer-cli.git'
+def git_ref = 'master'
+def package_name = 'rubygem-hammer_cli'
```

Comment:
I'm not too happy about this naming because on Debian it's named `ruby-hammer-cli`.

Reply:
While the `package_name` isn't used in the Debian branch of the job, I'd also prefer to call this `rpm_package_name`

Reply:
It's also not really the rpm package name since that could be `tfm-rubygem-hammer_cli`. `obal_name` or `obal_manifest_name` perhaps?

Reply:
`packaging_folder_name` ?

Reply:
folder is typically a windows name where unix uses directory. `obal_package_name` perhaps? Currently this line needs an update anyway